README.md

```diff
@@ -0,0 +1,96 @@
+# Conduit Connector Google Cloud Platform Pub/Sub
+
+### General
+The GCP Pub/Sub connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. 
+It provides both, a source and destination GCP Pub/Sub connector.
+
+The connector supports [Pub/Sub Lite](https://cloud.google.com/pubsub/lite/docs) service.
+To use the Pub/Sub Lite service, it is necessary to fill out the `location` configuration field.
+
+### Prerequisites
+- [Go](https://go.dev/) 1.18
+- (optional) [golangci-lint](https://github.com/golangci/golangci-lint) 1.46.2
+
+### How to build it
+Run `make build`.
+
+### Testing
+Run `make test`.
+
+### How it works
+Under the hood, the connector uses [Google Cloud Client Libraries for Go](https://github.com/googleapis/google-cloud-go).
+
+### Source
+A source connector represents the receiver of the messages from the GCP Pub/Sub or Pub/Sub Lite services.
+
+#### How it works
+The system contains two queues in memory.
+
+The first queue contains records witch were returned by the `Read` method.
+Messages are continuously added to this queue as soon as they appear in the topic.
+
+The second queue exists to acknowledge records using the `Ack` method. 
+Messages are added to this queue immediately after a record is returned by the Read method.
+
+If new messages are sent to GCP Pub/Sub while the connector is down, 
+these messages will be received after the connector is up.
+
+**CDC**: Messages that are in the service cannot be deleted or changed. 
+Consequently, all messages have no `action` key in the metadata.
```

Comment:
Is this still relevant, because we are now using OpenCDC?

Reply:
Yep. It's not relevant for now.
Thank you!